mm2src/mm2_event_stream/src/streamer.rs

```diff
@@ -16,6 +19,47 @@ pub trait StreamHandlerInput<D>: Stream<Item = D> + Send + Unpin {}
 /// Implement the trait for all types `T` that implement `Stream<Item = D> + Send + Unpin` for any `D`.
 impl<T, D> StreamHandlerInput<D> for T where T: Stream<Item = D> + Send + Unpin {}
 
+#[derive(Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
```

Comment:
Please keep attributes in alphabetical order. This convention is followed in many parts of the repository to improve readability.